README.md

```diff
@@ -58,19 +58,19 @@ Madara is a powerfull Starknet hybrid client written in Rust.
    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s
    ```
 
-   Clone the Madara repository:
+   Clone the Deoxys Madara full node repository:
```

Comment:
could you keep the naming here

Reply:
we are migrating toward madara so anything that doesnt technically affect the code or its deployement should be under madara namings